res/controllers/Numark-Mixtrack-3-scripts.js

```diff
@@ -614,12 +603,15 @@ LongShortDoubleBtn.prototype.ButtonDown = function(channel, control, value, stat
         this.ButtonCount = 1;
         // and short press
         this.ButtonLongPress = false;
-        this.ButtonLongPressTimer = engine.beginTimer(this.LongPressThreshold, function() {
-            myself.ButtonAssertLongPress();
-        }, true);
-        this.ButtonTimer = engine.beginTimer(this.DoublePressTimeOut, function() {
-            myself.ButtonAssert1Press();
-        }, true);
+
+        this.ButtonLongPressTimer = engine.beginTimer(
+            this.LongPressThreshold, this.ButtonAssertLongPress, true
```

Comment:
Hmm, I think this actually would have worked in Mixxx 2.0. What #1196 fixed was using `this` inside the callback function. For example: fd9815dd042e1a33b717d49db067271cef458c96